components/calendar/src/types.rs

```diff
@@ -26,12 +26,22 @@ use zerovec::ule::AsULE;
 #[allow(clippy::exhaustive_structs)] // this is a newtype
 pub struct Era(pub TinyStr16);
 
-/// Representation of a formattable year.
+impl Era {
+    /// Construct an unknown era for GIGO behavior during
+    pub const fn unknown() -> Self {
+        Era(tinystr::tinystr!(16, "unknown"))
+    }
+}
+
+/// General information about a year, needed by Temporal.
 ///
 /// More fields may be added in the future for things like extended year
 #[derive(Copy, Clone, Debug, PartialEq)]
 #[non_exhaustive]
-pub struct FormattableYear {
+pub struct YearInfo {
```

Comment:
Issue: Temporal does not _require_ the era field. It just does not currently know about cycles.

Citation: https://tc39.es/proposal-temporal/#table-temporal-calendar-date-record-fields